eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -306,6 +306,20 @@ jobs:
         # we want to run this when TestFromSource isn't true
         condition: and(succeeded(), ne(variables['TestFromSource'],'true'))
 
+      - task: Maven@3
+        displayName: 'Generate aggregate code coverage report'
+        condition: and(eq(variables['OsName'], 'Windows'), eq(variables['JavaTestVersion'], '1.11'))
```

Comment:
Do we only want to run this on Windows or Linux? Linux would run much faster, if that is a concern.

Reply:
Well, only one job can publish the report. If multiple jobs publish, the last one wins and invariably the "Build from source" job that runs on Linux and JDK 11 will be the last and publishes empty results. Given that this is only collecting report from already created jacoco reports from individual modules, speed is not a huge concern and Windows or Linux doesn't make a huge difference.